mir_eval/separation.py

```diff
@@ -442,6 +443,102 @@ def bss_eval_images(reference_sources, estimated_sources,
         return (sdr, isr, sir, sar, popt)
 
 
+def bss_eval_images_framewise(reference_sources, estimated_sources,
+                              window=30*44100, hop=15*44100,
+                              compute_permutation=False):
+    """Framewise computation of bss_eval_images
+
```

Comment:
Okay, I added the first as a reference in the `bss_eval_sources` and `bss_eval_images` docstrings.